build.gradle

```diff
@@ -0,0 +1,56 @@
+buildscript {
+	ext {
+		springBootVersion = '1.5.1.RELEASE'
+	}
+	repositories {
+		mavenCentral()
+	}
+	dependencies {
+		classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
+	}
+}
+
+plugins {
+    id "com.gorylenko.gradle-git-properties" version "1.4.16"
```

Comment:
This plugin adds GIT info about the app when building.  This information shows up on the /info actuator endpoint - http://localhost:8081/manage/info